src/Microsoft.AspNetCore.Diagnostics.Elm/baseline.netframework.json

```diff
@@ -0,0 +1,2210 @@
+{
```

Comment:
IIRC correctly API check looks for `baseline.netcore.json` when the project targets .NET Standard, so this file needs to be renamed.